### PR TITLE
Feature/dev.iss49636.tac2 bufr upper air

### PR DESCRIPTION
**Pull request by ChristopherHill-NOAA**
Control scripts for GFS/GDAS, CDAS, and RAP are modified to enable BUFR sonde data dump processing.  Testing is needed for each of the scripts modified.  This development branch was based from the first release branch to WCOSS2.

### CODE REVIEW

File: scripts/exrap_dump.sh [hunk 1]
```diff
@@ -68,9 +69,14 @@ set +u
 # ------------------------------------------------------------------------
 # Dump group #1 (non-pb) = 1bamua 1bmhs esamua esmhs atms mtiasi sevcsr
 #                          gpsro esiasi iasidb esatms atmsdb sevasr amsr2
+<<<<<<< HEAD
```
Comment (ShelleyMelchior-NOAA) [inline scripts/exrap_dump.sh:72]: Need to remove or reconcile these merge conflict notations

File: scripts/exrap_dump.sh [hunk 2]
```diff
@@ -68,9 +69,14 @@ set +u
...
+# Dump group #2 (pb) = vadwnd satwnd adpupa uprair
+# Dump group #3 (pb) = proflr rassda sfcshp adpsfc ascatt tideg
+=======
```
Comment (ShelleyMelchior-NOAA) [inline scripts/exrap_dump.sh:75]: Need to remove or reconcile these merge conflict notations

File: scripts/exrap_dump.sh [hunk 3]
```diff
@@ -68,9 +69,14 @@ set +u
...
 #                          subpfl saldrn
 # Dump group #2 (pb) = vadwnd satwnd adpupa
 # Dump group #3 (pb) = proflr rassda sfcshp adpsfc ascatt tideg snocvr
+>>>>>>> origin/release/obsproc.v1.1.0
```
Comment (ShelleyMelchior-NOAA) [inline scripts/exrap_dump.sh:79]: Need to remove or reconcile these merge conflict notations

**Review by ShelleyMelchior-NOAA — CHANGES_REQUESTED** [review]
The only problems I see are in exrap_dump.sh. There are some conflict notations that need to be removed/reconciled.

**Comment by ilianagenkova** [comment]
@nicholasesposito , when you are back and ready to test Chris's branches, please consider @ShelleyMelchior-NOAA's review. thanks!

**Comment by nicholasesposito** [comment]
Back today and will do @ilianagenkova

File: scripts/exrap_dump.sh (hunk 1 above)
Comment (ilianagenkova) [reply scripts/exrap_dump.sh:72]: I resolved this conflict in feature/dev.iss49636.TAC2BUFR_upper-air.v1.1

**Comment by ilianagenkova** [comment]
@nicholasesposito , @ShelleyMelchior-NOAA suggestions are implemented in feature/dev.iss49636.TAC2BUFR_upper-air.v1.1/ (which is simply @ChristopherHill-NOAA 's branch + release/obsproc.v1.1 new bits)

**Pull request closed without merging**